Source/engine.cpp

```diff
@@ -248,6 +252,24 @@ int32_t GenerateRnd(int32_t v)
 	return AdvanceRndSeed() % v;
 }
 
+void SetRndSeed_rngv2(int32_t s)

```

Comment:
`SetRndSeed_rngv2` isn't proper camel case and a bit redundant. Could cal it `SetRndSeedV2` or `SetStrongRndSeed`

Reply:
well it's fixable with 1 global replace, my goal was a name that was unique enough so a search would return only my stuff instead of 100k stuff from asio/whatever xD

Reply:
> well it's fixable with 1 global replace

Do it :)